Documentation/ContentObjects/Pageview/Index.rst

```diff
@@ -158,6 +157,10 @@ Properties
         sub-property :typoscript:`options` can be used to pass parameters to the
         processor class.
 
+        It is recommended to use the  :ref:`PageContentFetchingProcessor <PageContentFetchingProcessor>`
```

Comment:
If it is recommended I prefer to extend the linked example accordingly

Reply:
The example displayed here uses that data processor?

Reply:
This is the current original:
https://docs.typo3.org/m/typo3/reference-typoscript/main/en-us/ContentObjects/Pageview/Index.html#confval-pageview-dataprocessing

The linked example goes to here:
https://docs.typo3.org/m/typo3/reference-typoscript/main/en-us/ContentObjects/Pageview/Index.html#cobj-pageview-dataProcessing-example

This example does not contain the recommended DataProcessor.

What do you think? Keep it as it is? Update the linked example? Update the link to the better and new examples of your PR

Reply:
That example does not yet include information on how to display the content elements. as displaying the content elements is incomplete without https://review.typo3.org/c/Packages/TYPO3.CMS/+/83718/2 beeing merged I would keep the other example as-is for now